app/views/shared/page_templates/_default_page_template.html.erb

```diff
@@ -8,9 +8,7 @@
         <%= content_for(:page_title) %>
       </h3>
```

Comment:
I think new `page_heading` method can be used here, but I didn't make that change in this PR but rather have postponed that change to when a page including a panel is being updated, so that the change can be properly tested.